build_system/src/main.rs

```diff
@@ -26,16 +26,30 @@ macro_rules! arg_error {
 fn usage() {
     println!(
         "\
-Available commands for build_system:
+        Usage: build_system [command] [options]
 
-    cargo     : Run cargo command
-    clean     : Run clean command
-    prepare   : Run prepare command
-    build     : Run build command
-    test      : Run test command
-    info      : Run info command
-    clone-gcc : Run clone-gcc command
-    --help    : Show this message"
+    Available commands for build_system:
+
+        cargo     : Executes a cargo command. Use 'cargo --help' for a list of cargo commands.
```

Comment:
Please remove `Use 'cargo --help' for a list of cargo commands.` here and below since `--help` is already mentionned in the `Options` section below.

Perhaps it would make it more obvious that this is available and possible if you moved the `Options` section above the command section?

Reply:
i see, that makes more sense